### ISSUE TITLE
Fix specificity of Customer/Project identifiers

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#424]
### TLDR ###
*Which identifier is more important: `Category`, `Title` or `Body`?* 

Ties in with discussion in #111 and subsequent PR #112 

### Discussion ###
Let's say I hypothetically get invited to an event titled `Bid 360 weekly status`.
This event has `[IGNORE]` in the body.
I don't want to change the body text, since my changes will be lost if the organizer sends an update.
I add a `PMI BID360` category to the event, and expect the event to show up in Did.
It doesn't, so I'll have to create a separate event for logging the actual meeting hours.

I realise this only happens with the `[IGNORE]` tag, but from an end user perspective, I'd expect the following order of specificity (rated most to least important):

1. Any matching category. categories are Dids **!important** statements IMO.
2. [IGNORE] string in the title or body (category covered by 1.)
3. Customer + Project tag in title or body (category already covered by 1.)
4. optional/future: Partial match on title
